server/controllers/refer/referralsController.test.ts

```diff
@@ -92,6 +98,29 @@ describe('ReferralsController', () => {
         courseOfferingId,
         pageHeading: "Enter the person's identifier",
       })
+
+      expect(FormUtils.setFieldErrors).toHaveBeenCalledWith(request, response, ['prisonNumber'])
+      expect(response.locals.errors).toEqual(emptyErrorsLocal)
+    })
+
+    describe('when there are form errors', () => {
```

Comment:
Let's remove this test from the controller, as it's more a test of `setFieldErrors`'s functionality. I think all that's important to test here is that the (mock) function is called with the correct arguments (which we do above). We can test this behaviour in our integration tests.

This should also reduce some of the overhead when adding tests for errors in future.

Reply:
Agree - I did pause on these two tests wondering which should do what/whether either should do some of these assertions